auth.go

```diff
@@ -444,6 +452,39 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 	return sessionkey, token, nil
 }
 
+func auditUserAction(identity, password, actionDescription string) error {
+	auditServiceUrl := "http://localhost:2016/auth-logaction"
+	client := &http.Client{}
+
+	currentTime := time.Now().Unix()
+	currentLocation := time.Now().Location().String()
+
+	actionStr := []byte(fmt.Sprintf(`{"who": "User %v with password %v","did_what": "%v", "to_what": "IMQS Web V8", "at_time": %v, "context": {"location": "%v"}}`, identity, password, actionDescription, currentTime, currentLocation))
```

Comment:
Rather use a struct and marshall it to JSON. Doing inline JSON like this is fine for very very small things, like a ping response, but it's going to lead to trouble when you accept arbitrary strings as input. For example, if any of the input strings had a double quote, you would have invalid JSON. Just use json.marshal, it's much neater.